src/D2L.Bmx/Program.cs

```diff
@@ -116,32 +120,41 @@
 
 rootCommand.Add( printCommand );
 
-var writeCommand = new Command( "write", "Write to AWS credentials file" )
-	{
-		accountOption,
-		durationOption,
-		orgOption,
-		writeOutputOption,
-		profileOption,
-		roleOption,
-		userOption,
-	};
+var writeCommand = new Command( "write", "Write to AWS credentials file" ) {
+	accountOption,
+	durationOption,
+	orgOption,
+	writeOutputOption,
+	profileOption,
+	roleOption,
+	userOption,
+	nonInteractiveOption,
```

Comment:
the write command doesn't currently accept the `--non-interactive` flag, but I think it should for consistency.